core/src/free/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_cast_message">@string/pref_cast_message_free_flavor</string>
+</resources>
```

Comment:
@mfietz What do you mean by this?

I referred to strings defined on the main strings.xml because I assume it would otherwise complicate the translations.